app/src/main/java/com/example/drawingApp/EditLayerAdapter.kt

```diff
@@ -0,0 +1,76 @@
+package com.example.drawingApp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+
+class EditLayerAdapter(
+    private val data: List<LayerViewModel>,
```

Comment:
I did a double take here, but I assume this is a LayerViewModel since you are expecting it to hold a bitmap at some point

Mine is currently a list of strings, but I like the forward thinking.